jest.config.js

```diff
@@ -0,0 +1,187 @@
+// For a detailed explanation regarding each configuration property, visit:
```

Comment:
@esakal can we can rid of all comments in this file?

Reply:
sure